# object pose meta information

SinglePoseInfo[] meta_info